deployment/helm/node-feature-discovery/templates/worker.yaml

```diff
@@ -54,6 +54,14 @@ spec:
         - "-key-file=/etc/kubernetes/node-feature-discovery/certs/tls.key"
         - "-cert-file=/etc/kubernetes/node-feature-discovery/certs/tls.crt"
 {{- end }}
+        {{- if .Values.metrics.enable }}
+        - "-metrics={{ .Values.metrics.port }}"       
+        {{- end }}
```

Comment:
I think this doesn't work as expected as `--metrics` is enabled in nfd-master (in port 8081) by default(?)

If we want the option to disable, maybe we should have `.Values.metrics.enable: true` by default and the set `--metrics=0` if it's false?